### ISSUE TITLE
Missing semicolon incorrectly reported as final newline issue

### ISSUE THREAD

**Issue by royaldark** [sasstools/sass-lint#967]
**What version of Sass Lint are you using?**
1.10.2

**Please include any relevant parts of your configuration**
I am using the default config, with `final-newline` and `trailing-semicolon` enabled.

**What did you do? Please include the actual source code causing the issue.**
```scss
@import 'another-file'
 
```

Notice there *is* a newline after the `@import` line.

**What did you expect to happen?**
Triggers the `trailing-semicolon` rule, since I forgot it on `@import`.

**What actually happened? Please include any error messages given to you by Sass Lint.**
Triggers the `final-newline` rule, which is incorrect. By adding a semicolon, no rules are triggered.

```
1:23  warning  Files must end with a new line  final-newline
```

Note that this is not unique to a this exact setup. A file of any length which ends with an `@import` statement without a trailing semicolon reproduces the issue.

**Comment by DanPurdy** [comment]
I'll have to check this but definitely a bug.
